sites/website/src/docs/resources/cheat-sheet.md

```diff
@@ -0,0 +1,711 @@
+| fromView |  This mode skips reflecting the value of the property back to the HTML attribute.
+
+---
+
+Supply a custom `ValueConverter` by setting the converter property of the attribute configuration:
+
+```ts
+interface ValueConverter {
+    toView(value: any): string;
+    fromView(value: string): any;
+}
+```
+
+Example:
+
+```ts
+const numberConverter: ValueConverter = {
+  toView(value: number): string {
+    return String(value);
+  },
+
+  fromView(value: string): number {
+    return Number(value);
+  }
+};
+
+@customElement('my-counter')
+export class MyCounter extends FASTElement {
+  @attr({ mode: 'reflect', converter: numberConverter }) count: number = 0;
+}
+```
+
+---
+### [Templates](https://www.fast.design/docs/fast-element/declaring-templates)
+
+To create an HTML template for an element, import and use the html tagged template helper and pass the template to the `@customElement` decorator:
+
+```ts
+import { FASTElement, customElement, attr, html } from '@microsoft/fast-element';
+
+const template = html<NameTag>`
+  <div class="header">
+    <h3>${x => x.greeting.toUpperCase()}</h3>
+    <h4>my name is</h4>
+  </div>
+
+  <div class="body">TODO: Name Here</div>
+
+  <div class="footer"></div>
+`;
+
+@customElement({
+  name: 'name-tag',
+  template
+})
+export class NameTag extends FASTElement {
+  ...
+}
+```
+
+---
+
+
+### [Observables](https://www.fast.design/docs/fast-element/observables-and-state#observable-features)
+
+To enable binding tracking and change notification, properties must be decorated with either `@attr` or `@observable`. 
+
+Use `@attr` for primitive properties (string, bool, number) that are intended to be surfaced on your element as HTML [attributes](#attributes). Use `@observable` for all other property types on an HTMLElement and all observable properties on plain classes.
+
+
+```ts
+import { Observable } from '@microsoft/fast-element';
+
+export class Person {
+  @observable firstName = '';
+  @observable lastName = '';
+
+  get fullName() {
+    return `${this.firstName} ${this.LastName}`;
+  }
+}
+```
+
+---
+### [Bindings](https://www.fast.design/docs/fast-element/declaring-templates#understanding-bindings)
+
+| Binding Type | Example | Notes |
+| :--- | :--- | :--- |
+| Content | `<a>${…}</a>` | Creates a binding to interpolate text or child templates into element content. |
+| HTML Attribute | `<a href=${…}></a>` | Creates a binding that uses the setAttribute API. Attribute bindings also support interpolation with text and other bindings. |
+| HTML Boolean Attribute | `<input ?disabled=${…}>` | Creates a binding that adds or removes the attribute based on truthy/falsey values. |
+| JS Property | `<input :value=${…}>` | Creates a binding that sets a JavaScript property on the element. |
+| Event Handler | `<button @click=${…}>...</button>` | Registers an event handler using addEventListener. The listener is automatically removed when the template is unbound. |
```

Comment:
We should mention that events bound this way are automatically marked with preventDefault unless the event handler returns 'true'.

Reply:
@scomea Thank you - updated!